README.md

```diff
@@ -30,6 +19,14 @@ These scripts simplify the process of:
 
 New user data is assumed to be in a Google Sheet. The function parse_rows in addusers.py handles parsing the spreadsheet data, you may need to modify it to work with your particular spreadsheet format.
 
+## Usage
+To run it:
+```
+# Possibly in a virtual environment 
+$ pip install -r requirements.txt 
```

Comment:
^I forgot which repo I was reviewing here but this is still true - please update the README to show that the pip install command is a setup task.

The `python -m` below in line 27 is not needed.  That is used in setpass to start the test server.  Here we do not have a test server.